aws-cli/bash-linux/s3/bucket-lifecycle-operations/README.md

```diff
@@ -0,0 +1,39 @@
+<!--
+All content in this repository, unless otherwise stated, is Copyright © Amazon Web Services, Inc. or its affiliates. All rights reserved.
+
+Except where otherwise noted, all examples in this collection are licensed under the Apache license, version 2.0 (the "License"). The full license text is provided in the LICENSE file accompanying this repository.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+-->
+
+# Amazon S3 Bucket Lifecycle Operations
+
+## Purpose
+
+This example demonstrates how to interact with some of the basic Amazon S3 operations. The script includes functions that perform the following tasks:
```

Comment:
Which script? There are three. You might start out with a name/description for all three. Just a one line description for each so the reader has a basic understanding of their relationships.